CHANGELOG.md

```diff
@@ -116,7 +116,7 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.2.0 - TBD
+## v2.2.0 - 2022-09
```

Comment:
we should also add the `## v2.3.0 - TBD` right?

Reply:
you could also do it in the version bump PR.

Reply:
I will do that in the PR bumping new versions.